internal/provider/mw.go

```diff
@@ -9,12 +9,14 @@ import (
 	"github.com/threefoldtech/terraform-provider-grid/pkg/subi"
 )
 
+const ErrTerraformOutSync = "Error reading data from remote, terraform state might be out of sync with the remote state"
```

Comment:
Not sure if we need to export the error at this point, but if so we can add comments to the constant